modules/LoginPage/containers/LoginContainer.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { Box, Flex, Image } from "@mantine/core";
+
+import LoginForm from "../components/LoginForm/LoginForm";
+
+function LoginContainer() {
```

Comment:
```suggestion
const LoginContainer = () => {
```
Please create a ticket to convert all component definitions into arrow functions.